stories/Icons.story.tsx

```diff
@@ -0,0 +1,325 @@
+import * as React from 'react';
+
+import {
+    description,
+    importExample,
+    header,
+    divider,
+    tab,
+    tabs,
+    title,
+    code as baseCode
+} from 'wix-storybook-utils/Sections';
+
+
+import { allComponents } from './utils/allComponents';
+
+const code = config =>
+    baseCode({ components: allComponents, compact: true, ...config });
+
+export default {
+    category: 'Icons',
+    storyName: 'Icons',
+    sections: [
+        header(),
+        tabs([
+            tab({
+                title: 'Icon Inventory',
+                sections: [
+                    description({
+                        title: 'Importing the icons',
+                        text: [
+                            'We are using the ',
+                            <a href="https://www.npmjs.com/package/@svgr/webpack" >
+                                svgr plugin
+                            </a>,
+                            ' for svg imports which allow you to import the svg file directly. ',
+                            'If you have ',
+                            <a href="https://webpack.js.org/guides/tree-shaking/" >
+                                Tree Shaking
+                            </a>,
+                            ' in your application use the first option otherwise use the second option.'
+                        ],
+                    }),
+
+
+                    importExample(
+                        "import { Icons } from 'wix-ui-tpa';\n" +
+                        "<Icons.Calendar />",
+                    ),
+
+                    importExample(
+                        "import { ReactComponent as Calendar } from 'wix-ui-tpa/dist/src/assets/icons/Calendar.svg';\n" +
+                        "<Calendar />",
+                    ),
```

Comment:
@shlomitc - Can you pls verify that I explained the 2 ways for importing our icons correct?

Reply:
I don't think we want to start changing the library at the moment.
I think it's best to focus on what we have at the moment, and change the doc once we change the library.
There are a lot of changes that we're going to make to the icons after the design team will finish working on the new icon inventory, so let's not mix happiness with happiness ?